xbmc/utils/URIUtils.h

```diff
@@ -28,9 +28,6 @@ class CURL;
 class URIUtils
```

Comment:
Missing `final`?

Reply:
Hm, better convert to `namespace`. This isn't a class after all - it just abuses the `class` keyword to create a namespaceo.

Reply:
+1

`StringUtils` is similar, but you don't have to change either one in this PR.